packages/dd-trace/src/config.js

```diff
@@ -945,6 +946,7 @@ class Config {
     this.#setString(target, 'version', DD_VERSION || tags.version)
     this.#setBoolean(target, 'inferredProxyServicesEnabled', DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED)
     this.#setBoolean(target, 'trace.aws.addSpanPointers', DD_TRACE_AWS_ADD_SPAN_POINTERS)
+    this.#setBoolean(target, 'trace.ws.addSpanPointers', DD_TRACE_WS_ADD_SPAN_POINTERS)
```

Comment:
There should not be an additional configuration. If WS is enabled then span pointers is enabled.